### ISSUE TITLE
Copy/paste mesh results from terminal into visualizer

### ISSUE THREAD

**Issue by amooz** [jneilliii/OctoPrint-BedLevelVisualizer#209]
It would be great if we could copy/paste the mesh results from the terminal logs into a text editor and update the visualization that way.  If there's a problem scraping the info from the terminal logs, we would still be able to update the visualization (just with some extra steps.)

**Comment by jneilliii** [comment]
You run into some of the same issues relative to parsing because of so many of the different variants of the report from the different firmware and versions of firmware.

**Comment by amooz** [comment]
As a first-pass, what do you think of making this an advanced feature and you only accepted a CSV.  It would require some parsing on the users end

**Comment by tablatronix** [comment]
How can we just get mesh data without probing to update it?

**Comment by jneilliii** [comment]
I assume you mean with something like manual mesh leveling?

**Comment by tablatronix** [comment]
Like `M420 S1 V` to get mesh from printer not recreate another one. 

Is this the data you mean is hard to parse?

**Comment by jneilliii** [comment]
Yeah, that's pefect. You can enter that in the plugin's setting (I use `M420 V` personally with UBL because I don't probe every print). You can even use custom controls to achieve multiple slots, additional discussion [here](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/issues/375).

**Comment by tablatronix** [comment]
Ahh I did not realize custom commands could link to the grapher nice gonna test now
